scrnaseq.Rmd

```diff
@@ -1030,9 +1032,31 @@ if (length(p_list)==1) {
 }
 p
 ```
+```{r downsampling}
+# If requested, determine number of cells for downsampling
+if (is.null(param$downsample_cells_n) & param$downsample_cells_equally) {
+  n = purrr::map_int(sc, ncol) %>% min()  
+} else if (!is.null(param$downsample_cells_n)) {
+  n = param$downsample_cells_n
```

Comment:
What will happen if a sc object has less than n cells?

Reply:
You take care of that in `ScSampleCells`: 
```
  # Set n
  if (n > ncol(sc)) n = ncol(sc)
  ```

Reply:
yes true